src/main/flight/pid.c

```diff
@@ -145,9 +145,9 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .throttle_boost_cutoff = 15,
         .iterm_rotation = false,
         .smart_feedforward = false,
-        .iterm_relax = ITERM_RELAX_OFF,
-        .iterm_relax_cutoff_low = 3,
-        .iterm_relax_cutoff_high = 30,
+        .iterm_relax = ITERM_RELAX_RPY,
+        .iterm_relax_lowpass = 10,
+        .iterm_relax_cutoff = 60,
```

Comment:
Could you explain why you change the default settings ?
I'm still unable to understand how they work ...